<launch>

  <!-- launch-prefix="xterm -e gdb -ex run -args" -->
  <node pkg="hba" type="hba" name="hba" output="screen"  >
    <param name="data_path" type="string" value="/home/map/region2-65/"/>
    <param name="total_layer_num" type="int" value="2"/>
    <param name="pcd_name_fill_num" type="int" value="6"/> <!--set 5 for kitti07 and 0 for park-->
    <param name="thread_num" type="int" value="6"/>
  </node>

  <arg name="rviz" default="0"/>
  <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz_hba"
    args="-d $(find hba)/rviz_cfg/debug.rviz" />
  </group>
</launch>